src/app/phast/losses/energy-input-exhaust-gas-losses/energy-input-exhaust-gas-losses.module.ts

```diff
@@ -1,16 +1,23 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { ReactiveFormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { EnergyInputExhaustGasFormComponent } from './energy-input-exhaust-gas-form/energy-input-exhaust-gas-form.component';
 import { EnergyInputExhaustGasLossesComponent } from './energy-input-exhaust-gas-losses.component';
 import { EnergyInputExhaustGasCompareService } from './energy-input-exhaust-gas-compare.service';
 import { EnergyInputExhaustGasService } from './energy-input-exhaust-gas.service';
 import { SharedPipesModule } from '../../../shared/shared-pipes/shared-pipes.module';
+import { FlueGasModule } from '../../../calculator/furnaces/flue-gas/flue-gas.module';
+import { ModalModule } from 'ngx-bootstrap';
+import { SuiteDbModule } from '../../../suiteDb/suiteDb.module';
 @NgModule({
   imports: [
     CommonModule,
     ReactiveFormsModule,
-    SharedPipesModule
+    SharedPipesModule,
```

Comment:
The only imports needed are FlueGasModule and ModalModule. FlueGasModule imports the other modules, uses what it needs from those APIs and you get it all for free